code/backend/batch/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -25,28 +25,26 @@ def __init__(
             self.account_name = (
                 account_name if account_name else env_helper.AZURE_BLOB_ACCOUNT_NAME
             )
+            self.account_key = None
```

Comment:
Technically I didn't need to change this file, but I just reordered it slightly to make it more obvious, how initialisation for rbac is different from keys